src/test/java/tests/BaseTestAddRemoveElements.java

```diff
@@ -0,0 +1,30 @@
+package tests;
+
+import constants.Urls;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import utils.PropertyManager;
+
+public class BaseTestAddRemoveElements {
+    WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+        PropertyManager propertyManager = new PropertyManager();
+        propertyManager.loadData();
+        System.setProperty("webdriver.chrome.driver", propertyManager.get("PATH_TO_DRIVER"));
+        driver = new ChromeDriver();
+        driver.get(Urls.ADD_REMOVE_ELEMENTS_URL);
+        driver.findElement(By.xpath("//button[@onclick='addElement()']")).click();
+        driver.findElement(By.xpath("//button[@onclick='addElement()']")).click();
+        driver.findElement(By.xpath("//button[@onclick='deleteElement()']")).click();
+    }
+    @AfterMethod(alwaysRun = true)
+    public void tearDown() {
+        driver.quit();
+    }
```

Comment:
Please use 1 common base test for all tests

Reply:
done